Wikipedia/Code/ViewControllerRouter.swift

```diff
@@ -76,6 +76,17 @@ class ViewControllerRouter: NSObject {
                 return false
             }
             return presentOrPush(talkPageVC, with: completion)
+        case .onThisDay(let indexOfSelectedEvent):
+            let dataStore = MWKDataStore.shared()
+            guard let contentGroup = dataStore.viewContext.newestGroup(of: .onThisDay), let onThisDayVC = contentGroup.detailViewControllerWithDataStore(dataStore, theme: theme) as? OnThisDayViewController else {
```

Comment:
@mcleinman just thought of one other thing - could this pull a non-EN Wikipedia on this day group too, since you're only checking for the latest? Maybe we can add a language check here too. I can look into this later today.

Reply:
From changing the language of the app, this seems to work as expected.